frontend/src/components/DetailsTable.test.tsx

```diff
@@ -16,8 +16,63 @@
 
 import * as React from 'react';
 
-import DetailsTable from './DetailsTable';
-import { shallow } from 'enzyme';
+import DetailsTable, { DetailsFieldValue } from './DetailsTable';
+import { shallow, mount } from 'enzyme';
+
+describe('DetailsFieldValue', () => {
+  it('does render string as SPAN', () => {
+    const tree = shallow(<DetailsFieldValue index={1} fieldname={'key'} fieldvalue={'text'} />);
+    expect(tree).toMatchSnapshot();
```

Comment:
FYI, new snapshot testing guideline is to always use `toMatchSnapshotIneline()` and avoid taking long snapshots.

If several snapshots share a lot of things in common, take advantage of snapshot-diff to reduce duplication: e.g. https://github.com/kubeflow/pipelines/blob/5162e455a8571c55d6af5a995c835b33e2d8751c/frontend/src/components/viewers/Tensorboard.test.tsx#L61

For details: https://github.com/kubeflow/pipelines/pull/3166